docs/registry/index.tsx

```diff
@@ -3,7 +3,22 @@
 // Do not edit this file directly.
 import dynamic from 'next/dynamic';
 
-export const registry = {
+interface Registery {
```

Comment:
this is not coherent with the `main` branch. please use the script to update the file :)

Reply:
oh i got it